test/validation/api/ipsec/ipsec.c

```diff
@@ -935,21 +935,18 @@ static void parse_ip(odp_packet_t pkt)
 	(void)odp_packet_parse(pkt, l3, &param);
 }
 
-void ipsec_check_out_one(const ipsec_test_part *part, odp_ipsec_sa_t sa)
+int ipsec_check_out(const ipsec_test_part *part, odp_ipsec_sa_t sa,
+		    odp_packet_t *pkto)
 {
-	int num_out = part->num_pkt;
-	odp_packet_t pkto[num_out];
 	int i;
+	int num_out;
 
 	num_out = ipsec_send_out_one(part, sa, pkto);
 
 	for (i = 0; i < num_out; i++) {
```

Comment:
Do we expect part->num_pkt to be more than 1? If not, this is still ipsec_check_out_one, right? Or may be, we should have this function take an argument 'num_part' and have this function work on more than one part?

Reply:
As I wrote in the other response, ipsec_test_part is about one operation and one operation can have multiple packets as output and therefore part->num_pkt could, in princple, be greater than one. Currently there are not such test cases though.

Reply:
@JannePeltonen Agreed.